src/Symfony/Component/Security/Core/Event/VoteEvent.php

```diff
@@ -51,7 +61,7 @@ public function getAttributes(): array
         return $this->attributes;
     }
 
-    public function getVote(): int
+    public function getVote(): Vote
```

Comment:
This is a BC break for event listeners calling this getter.

Reply:
I wonder how this should be resolved. Perhaps I can add a temporary method to return the vote object, and for getVote I will just return int to avoid breaking things?

Reply:
I reverted to int, I added a temporary method to get the vote object. Hope it works for now.